doc/user/faq.md

```diff
@@ -30,19 +31,30 @@ directly control the compiler. However this is complicated, so normally Truffle
 uses Graal on your behalf to *partially evaluate* your AST interpreter into
 machine code.
 
+### What is GraalVM?
+
+The GraalVM is a platform for high-performance polyglot programming. The GraalVM
+is the easiest way to get and use TruffleRuby.
+
+See [Using GraalVM](using-graalvm.md).
+
+More concretely, the GraalVM is a modified version of the OracleJDK that
+includes Truffle, Graal, TruffleRuby and other languages running on the Truffle
+framework including JavaScript, Python and R.
+
 ### How do I get TruffleRuby?
 
-The easiest way to get a JVM with Graal is via the GraalVM, available from the
-Oracle Technology Network. This includes the JVM, the Graal compiler, and
-TruffleRuby, all in one package with compatible versions. You can get either the
-runtime environment (RE) or development kit (DK).
+The easiest way to get TruffleRuby is the GraalVM, available from the Oracle
+Technology Network. This includes the JVM, the Graal compiler, and TruffleRuby,
```

Comment:
I think we should link to the OTN download page here.

Reply:
Fixing.